source/screens/stage.lua

```diff
@@ -1,8 +1,11 @@
+import "player"
```

Comment:
I propose the following so we can get rid of the weird checks and the specific flag inside of the player class.

```suggestion

running = false

-- Screen lifecycle

function StageScreen:init()
	StageScreen.super.init(self)
end

function StageScreen:onAppear()
end

function StageScreen:onDisappear()
end

function StageScreen:update()
    if running == false then
        updateStageIntro()
    end

    gfx.sprite.update()
end

function updateStageIntro() 
    gfx.drawText("Stage Screen", 140, 100)
    gfx.drawText("Press A to start", 130, 130)

    if pd.buttonJustPressed(pd.kButtonA) then
        gfx.clear(1)
        addPlayer()
        running = true
    end
end

function addPlayer()
    Player(200, 120)
end
```